elk/extraction/prompt_loading.py

```diff
@@ -105,7 +166,10 @@ def load_prompts(
     # templates for any dataset in order to make sure we don't run out of prompts.
     for ds_string in dataset_strings:
         ds_name, _, config_name = ds_string.partition(" ")
-        prompters.append(DatasetTemplates(ds_name, config_name))
+
+        if combined_template_path == "":
```

Comment:
nitpick, but I guess 

`if not combined_template_path: `

is a little bit cleaner